.github/workflows/RELEASE.yaml

```diff
@@ -142,7 +142,7 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: bundle/mvn/default-bundle/
-          push: ${{ github.ref == 'refs/heads/main' }}
+          push: true #${{ github.ref == 'refs/heads/main' }}
```

Comment:
Do we want to do anything with this?

Reply:
Sorry, it was left there by mistake, fixed it with my latest commit